gnomad/utils/transcript_annotation.py

```diff
@@ -416,10 +489,152 @@ def perform_tx_annotation_pipeline(
         ),
         tx_ht,
         tissues_to_filter=tissues_to_filter,
+        tissues_to_exclude_from_mean=tissues_to_exclude_from_mean,
         vep_root=vep_root,
         vep_annotation=vep_annotation,
     )
 
+    if "alleles" not in additional_group_by:
+        tx_ht = tx_ht.key_by("locus", "transcript_id").distinct()
```

Comment:
is each value the same? `distinct` chooses one of the values at random to deduplicate, right?

Reply:
yeah, that's a valid question. As is, it's really just to support the creation of the base-level file, and in that case this does work fine, but in other cases where more than only `gene_symbol` is provided, it might not work. I will modify to something I think is correct, let me know what you think.